cmd/podsync/updater.go

```diff
@@ -78,6 +78,7 @@ func (u *Updater) Update(ctx context.Context, feedConfig *config.Feed) error {
 		})
 
 		episodePath := filepath.Join(feedPath, u.episodeName(feedConfig, episode))
```

Comment:
`episodePath` not used anymore? How it ends up in proper feed data directory?

Reply:
it's still used for checking episode size and whether it exists. 
the actual url of the episode is generated in the same way in `update.go:222`.